src/builds/builds.controller.spec.ts

```diff
@@ -1,12 +1,12 @@
 import { Test, TestingModule } from '@nestjs/testing';
+import { Build } from '@prisma/client';
```

Comment:
guess this is the problem
this comes from local files, not from library

Reply:
Which are those files? I cannot find anything exporting `Build` ..